keyboards/xd004/keymaps/default/keymap.c

```diff
@@ -0,0 +1,13 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    // 0: Base Layer
+    LAYOUT_all(KC_L, KC_O, KC_V, KC_E),
```

Comment:
❤️

Reply:
That's actually the default layout it shipped with. Mechanical keyboards, spreading the love 😄